### DISCUSSION TITLE
Shopify Functions

### DISCUSSION THREAD

**Discussion by smshahriar10** [Kyon147/laravel-shopify#269] [Q&A] [answered]
How can we integrate [Shopify Functions](https://shopify.dev/docs/apps/functions) to an existing app using this package?

**Comment by Kyon147** [comment, marked as answer]
You need to set up an extension in your app using the Shopify CLI. 

Functions only support web assembly so you are likely going to have to use one of their recommended languages like Rust or JS  https://shopify.dev/docs/apps/functions
